docs/ref-info/glossary-uid.md

```diff
@@ -151,6 +154,15 @@ import Link from '@docusaurus/Link';
 <dd>Each UID2 <a href="/docs/intro#participants">participant</a> using a server-side implementation has an API key (client key) and also a secret value associated with the key, called the client secret (API secret). The client secret is known only to the participant and the UID2 service.</dd>
 <dd>For details, see <a href="/docs/getting-started/gs-credentials">UID2 Credentials</a>.</dd>
 
+<dt><MdxJumpAnchor id="gl-client-server">Client-server integration</MdxJumpAnchor></dt>
+<dd>UID2 integrations can be entirely on the <a href="#gl-client-side">client side</a>, entirely on the <a href="#gl-server-side">server side</a>, or partially on the client side and partially on the server side (client-server).</dd>
+<dd>For example, in a client-server integration for a publisher, the UID2 token is generated on the server side and refreshed on the client side.</dd>
+
+<dt><MdxJumpAnchor id="gl-client-side">Client-side integration</MdxJumpAnchor></dt>
+<dd>UID2 integrations can be entirely on the client side, entirely on the <a href="#gl-server-side">server side</a>, or partially on the client side and partially on the server side (<a href="#gl-client-server">client-server</a>).</dd>
+<dd>in a client-side integration, all activities relating to UID2, such as generating and refreshing UID3 tokens, are performed on the client side.</dd>
+<dd>For example, in a client-side integration, advertisers generate UID2 tokens on the client side for tracking pixels, and publishers generate UID2 tokens on the client side for bidstream use, as well as refreshing the tokens.</dd>
```

Comment:
same suggestion: maybe link to docs/guides/integration-mobile-client-side and docs/guides/publisher-client-side docs/guides/integration-prebid-client-side ?

Reply:
Done. Thx much.